packages/server/src/main/java/software/uncharted/terarium/hmiserver/configuration/TerariumEnvPostProcessor.java

```diff
@@ -0,0 +1,37 @@
+package software.uncharted.terarium.hmiserver.configuration;
+
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.env.EnvironmentPostProcessor;
+import org.springframework.core.Ordered;
+import org.springframework.core.annotation.Order;
+import org.springframework.core.env.ConfigurableEnvironment;
+
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+
+@Order(Ordered.HIGHEST_PRECEDENCE)
```

Comment:
Wouldn't we want this to be the lowest precedence? e.g. runs last instead of runs first?

Reply:
In this case the `Order` isn't really needed but just forced as convention.  You're able to set multiple `EnvironmentPostProcessor` classes and then able to number them 1,2,3 etc if order is important. This just maps to Integer.Max